bytecomp/bytegen.ml

```diff
@@ -345,11 +345,11 @@ let comp_primitive p args =
   | Pfloatcomp Cgt -> Kccall("caml_gt_float", 2)
   | Pfloatcomp Cle -> Kccall("caml_le_float", 2)
   | Pfloatcomp Cge -> Kccall("caml_ge_float", 2)
-  | Pstringlength -> Kccall("caml_ml_string_length", 1)
-  | Pstringrefs -> Kccall("caml_string_get", 2)
-  | Pstringsets -> Kccall("caml_string_set", 3)
-  | Pstringrefu -> Kgetstringchar
-  | Pstringsetu -> Ksetstringchar
+  | Pstringlength | Pbyteslength -> Kccall("caml_ml_string_length", 1)
```

Comment:
This PR introduces specific runtime primitives for bytes in str.c.  To be coherent, Pbyteslength should call a new `caml_ml_bytes_length` primitive.  Same for Pbytesrefs/u.